### PR TITLE
Add documentation for the new theming mechanism

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: docs/manage-default-themes-and-styles.md [hunk 1]
```diff
@@ -1,32 +1,91 @@
...
 - [Configure Styles in Vue](https://surveyjs.io/form-library/documentation/get-started-vue#configure-styles)
 - [Configure Styles in React](https://surveyjs.io/form-library/documentation/get-started-react#configure-styles)
 - [Link SurveyJS Resources in Knockout](https://surveyjs.io/form-library/documentation/get-started-knockout#link-surveyjs-resources)
 - [Link SurveyJS Resources in jQuery](https://surveyjs.io/form-library/documentation/get-started-jquery#link-surveyjs-resources)
 
-## Customize Themes
+## Apply a Predefined Theme
 
-### Colors and Sizes
+Predefined themes are distributed as JSON objects that specify CSS variables and other theme settings. You can find a full list of predefined themes and their variations on GitHub: [survey-core/themes](https://github.com/surveyjs/survey-library/tree/master/src/themes). To apply a predefined theme, import its module or reference its script _after_ `survey-core`. The following code shows how to apply the Layered Dark Panelless theme by importing a module:
```
Comment (JaneSjs) [inline docs/manage-default-themes-and-styles.md:25]: I would suggest adding preview images to illustrate predefined themes.

File: docs/get-started-angular.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ npm install survey-angular-ui --save
 
 ## Configure Styles
 
-SurveyJS ships with the Modern and Default V2 UI themes illustrated below.
+SurveyJS Form Library ships with several predefined themes illustrated below and a flexible theme customization mechanism based upon CSS variables.
```
Comment (annnke) [inline docs/get-started-angular.md:46]:
```suggestion
SurveyJS Form Library is shipped with several predefined themes illustrated below and a flexible theme customization mechanism based on CSS variables.
```

File: docs/get-started-vue.md [hunk 3]
```diff
@@ -33,34 +33,33 @@ npm install survey-vue-ui --save
 
 ## Configure Styles
 
-SurveyJS ships with the Modern and Default V2 UI themes illustrated below.
+SurveyJS Form Library ships with several predefined themes illustrated below and a flexible theme customization mechanism based upon CSS variables.
 
 ![Themes in SurveyJS Form Library](images/survey-library-themes.png)
 
-Open the Vue component in which your survey will be and import a style sheet that implements the required theme.
+To add SurveyJS themes to your application, open the Vue component in which your survey will be and import the Form Library style sheet:
```
Comment (annnke) [inline docs/get-started-vue.md:40]:
```suggestion
To add SurveyJS themes to your application, open the Vue component that will render your survey and import the Form Library style sheet:
```

File: docs/manage-default-themes-and-styles.md [hunk 4]
```diff
@@ -1,32 +1,91 @@
...
+---
 # Themes & Styles
 
-This help topic describes how to add SurveyJS themes to your Angular, Vue, React, Knockout, or jQuery application and switch between them if you want to allow users to change the active theme.
+This help topic describes how to add SurveyJS UI themes to your Angular, Vue, React, Knockout, or jQuery application, switch between them, or create a custom theme for your survey.
 
 ## Add SurveyJS Themes to Your Application
 
-SurveyJS ships with the Modern and Default V2 UI themes illustrated below.
+SurveyJS Form Library ships with several predefined UI themes. You can see what they look like and switch between them in [Form Library demos](https://surveyjs.io/form-library/examples/nps-question/).
```
Comment (annnke) [inline docs/manage-default-themes-and-styles.md:11]:
```suggestion
SurveyJS Form Library is shipped with several predefined UI themes. You can see what they look like and switch between them in [Form Library demos](https://surveyjs.io/form-library/examples/nps-question/).
```

**Review by annnke — APPROVED** [review]

**Pull request merged**